WordPress/gutenberg
# Fix column moved in itself error

## Description
Closes: #13701
The bug is that empty column has no children and `dstClientId` was `undefined`, therefore it could not be recognized as moved block's child.

In solution I assumed that each 2 blocks that can be moved in each other are always separated by a `core/column` block. 

If that assumption is incorrect or unsustainable I can leave:
```jsx
  isSrcBlockAnAncestorOfDstBlock( srcClientId, dstClientId )
```
and just add:
```jsx
  isSrcBlockAnAncestorOfDstBlock( srcClientId, dstRootClientId )
```
along the previous version.

## How has this been tested?
1. Create new column block.
2. Drag it to second column.
3. See that it doesn't crash.

## Screenshots 

## Types of changes
Bug fix

## Checklist:
